beluga/test/beluga/include/beluga/test/static_occupancy_grid.hpp

```diff
@@ -31,6 +31,15 @@ class StaticOccupancyGrid : public BaseOccupancyGrid2<StaticOccupancyGrid<Rows,
     [[nodiscard]] bool is_free(bool value) const { return !value; }
     [[nodiscard]] bool is_unknown(bool) const { return false; }
     [[nodiscard]] bool is_occupied(bool value) const { return value; }
+    [[nodiscard]] beluga::GridStatus get_status(bool value) const {
```

Comment:
@DPR00 consider moving this over to the `BaseOccupancyGrid` class. The fewer the requirements on the user the better.